requirements.txt

```diff
@@ -0,0 +1,8 @@
+# YAML
+ruamel.yaml
+
+# Linting
+flake8
+
+pyalsaaudio
+pyzmq
```

Comment:
What's the difference of this file compared to this? 
* https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future1/src/jukebox/requirements.txt

Do we need to run `pip install` twice? Might be a Python newbie question, but just wanted to check if there are possibilities to combine.

Reply:
Didn't see. Github automatic run fails as top-level requirements.txt is missing.